tests/parsers/test_orca.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Tests for the :class:`aiida_orca.parsers.OrcaBaseParser` parser."""
+
+
+def test_orca_default(
+    aiida_localhost, generate_calc_job_node, generate_parser, generate_inputs_orca, data_regression, num_regression
+):
+    """Test the ``default`` output example."""
+    name = 'default'
+    entry_point_calc_job = 'orca.orca'
+    entry_point_parser = 'orca_base_parser'
+
+    node = generate_calc_job_node(entry_point_calc_job, aiida_localhost, name, generate_inputs_orca())
+    parser = generate_parser(entry_point_parser)
+    results, calcfunction = parser.parse_from_node(node)
+
+    assert calcfunction.is_finished, calcfunction.exception
+    assert calcfunction.is_finished_ok, calcfunction.exit_message
+    assert 'relaxed_structure' in results
+    assert 'output_parameters' in results
+
+    structure_attributes = results['relaxed_structure'].attributes
+    structure_cell = [item for sublist in structure_attributes.pop('cell') for item in sublist]
+
+    data_regression.check({
```

Comment:
Where do these fixtures come from? What is the intended use case for `data_regression` versus `num_regression`? (I mean I can guess but would like it to be spelled out explicitly)

Reply:
They come from the `pytest-regressions` package (https://pypi.org/project/pytest-regressions/). The `data_regression` is used for generic dictionary data structures whereas `num_regression` is used for numpy arrays: https://pytest-regressions.readthedocs.io/en/latest/api.html#num-regression

Reply:
Thank you for the explanation, that is very useful.